frontend/src/utils/YupSchema.tsx

```diff
@@ -48,23 +49,12 @@ export const Address = Yup.object().shape({
   postal: Yup.string().matches(/[a-zA-z][0-9][a-zA-z][0-9][a-zA-z][0-9]/, 'Invalid Postal Code'),
 });
 
+const currentYear = moment().year();
 export const Evaluation = Yup.object().shape({
-  fiscalYear: Yup.number()
-    .min(1900, 'Invalid Fiscal Year')
-    .max(2100, 'Invalid Fiscal Year')
-    .required('Required'),
-  estimatedValue: Yup.number()
-    .min(0, 'Estimated Value must be a positive number')
-    .required('Required'),
-  appraisedValue: Yup.number()
-    .min(0, 'Appraised Value must be a positive number')
-    .required('Required'),
-  assessedValue: Yup.number()
-    .min(0, 'Assessed Value must be a positive number')
-    .required('Required'),
-  netBookValue: Yup.number()
-    .min(0, 'Net Book Value must be a positive number')
-    .required('Required'),
+  year: Yup.number(),
+  date: Yup.string().required('Required'),
+  key: Yup.string().nullable(),
```

Comment:
Keys are not nullable.  How are you ensuring they are set to the correct enum value?

Reply:
Currently none of the selects are explicitly validated in Yup (mostly because most of them require an API call to get the list of accepted values).

Specific to this field, I'd argue that it should be removed entirely. The reason being that yup schema validation failures prevent a form from being submitted, and this field isn't visible to the user. So yup should be used to enforce user facing validation - other validation needs to be handled by other mechanisms.